### ISSUE TITLE
less4j 1.14.0 cause no viable alternative at input '[' in ruleset

### ISSUE THREAD

**Issue by M3lkior** [SomMeri/less4j#321]
Hi SomMeri,

I was using Wro4j wich brings less4j in v1.8.2. I recently updated Wro4j wich brings less4 in 1.14.0.
This version cause a regression in my less file

Here is my less style : 

```
.select2-dropdown-open.select2-drop-above [class ^= "select2-choice"] .btn-default {
	border: none ! important;
}
```

The exception : 
```
no viable alternative at input '[' in ruleset (which started at 335:43)
```

**Comment by SomMeri** [comment]
I tried it on latest less4j-1.15.2 and this bug was already fixed. 

There were [many bugfixes](https://github.com/SomMeri/less4j/wiki/Released-Versions) between 1.14.0 and now, so it might be a good idea to ask wro4j to upgrade again.

**Comment by M3lkior** [comment]
Yeah,

I will do :)

Thanks
